vgen/src/test/java/io/gapi/vgen/testdata/client_library.baseline

```diff
@@ -0,0 +1,84 @@
+============== file: library_service_client_config.json ==============
+{
+  "interfaces": {
+    "google.example.library.v1.LibraryService": {
+      "retry_codes": {
+        retry_codes_def:
+          "idempotent": [
+            "DEADLINE_EXCEEDED",
+            "UNAVAILABLE"
+          ],
+          "non_idempotent": []
+      },
+      "retry_params": {
+        "default": {
+          "initial_retry_delay_millis": 100,
+          "retry_delay_multiplier": 1.2,
+          "max_retry_delay_millis": 1000,
+          "initial_rpc_timeout_millis": 300,
+          "rpc_timeout_multiplier": 1.3,
+          "max_rpc_timeout_millis": 3000,
+          "total_timeout_millis": 30000
+        }
+      },
+      "methods": {
+        "create_shelf": {
+          "retry_codes_name": "non_idempotent"
+          "retry_params_name": "default"
+        }
+        "get_shelf": {
```

Comment:
FYI: JSON currently [cannot have comments](http://www.ipetitions.com/petition/allow-comments-in-json/)